test/e2e/common.go

```diff
@@ -46,35 +47,36 @@ import (
-	MultiTenancyIdentityName       = "MULTI_TENANCY_IDENTITY_NAME"
-	ClusterIdentityName            = "CLUSTER_IDENTITY_NAME"
-	ClusterIdentityNamespace       = "CLUSTER_IDENTITY_NAMESPACE"
-	ClusterIdentitySecretName      = "AZURE_CLUSTER_IDENTITY_SECRET_NAME"
-	ClusterIdentitySecretNamespace = "AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE"
-	AzureClientSecret              = "AZURE_CLIENT_SECRET"
-	AzureClientId                  = "AZURE_CLIENT_ID"
-	AzureSubscriptionId            = "AZURE_SUBSCRIPTION_ID"
-	AzureUserIdentity              = "USER_IDENTITY"
-	AzureIdentityResourceGroup     = "CI_RG"
-	JobName                        = "JOB_NAME"
-	Timestamp                      = "TIMESTAMP"
-	AKSKubernetesVersion           = "AKS_KUBERNETES_VERSION"
-	SecurityScanFailThreshold      = "SECURITY_SCAN_FAIL_THRESHOLD"
-	SecurityScanContainer          = "SECURITY_SCAN_CONTAINER"
-	ManagedClustersResourceType    = "managedClusters"
-	capiImagePublisher             = "cncf-upstream"
-	capiOfferName                  = "capi"
-	capiWindowsOfferName           = "capi-windows"
+	RedactLogScriptPath                   = "REDACT_LOG_SCRIPT"
+	AzureLocation                         = "AZURE_LOCATION"
+	AzureResourceGroup                    = "AZURE_RESOURCE_GROUP"
+	AzureVNetName                         = "AZURE_VNET_NAME"
+	AzureCustomVNetName                   = "AZURE_CUSTOM_VNET_NAME"
+	AzureInternalLBIP                     = "AZURE_INTERNAL_LB_IP"
+	AzureCPSubnetCidr                     = "AZURE_CP_SUBNET_CIDR"
+	AzureVNetCidr                         = "AZURE_PRIVATE_VNET_CIDR"
+	AzureNodeSubnetCidr                   = "AZURE_NODE_SUBNET_CIDR"
+	AzureBastionSubnetCidr                = "AZURE_BASTION_SUBNET_CIDR"
+	MultiTenancyIdentityName              = "MULTI_TENANCY_IDENTITY_NAME"
+	ClusterIdentityName                   = "CLUSTER_IDENTITY_NAME"
+	ClusterIdentityNamespace              = "CLUSTER_IDENTITY_NAMESPACE"
+	ClusterIdentitySecretName             = "AZURE_CLUSTER_IDENTITY_SECRET_NAME"
+	ClusterIdentitySecretNamespace        = "AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE"
+	AzureClientSecret                     = "AZURE_CLIENT_SECRET"
+	AzureClientId                         = "AZURE_CLIENT_ID"
+	AzureSubscriptionId                   = "AZURE_SUBSCRIPTION_ID"
+	AzureUserIdentity                     = "USER_IDENTITY"
+	AzureIdentityResourceGroup            = "CI_RG"
+	JobName                               = "JOB_NAME"
+	Timestamp                             = "TIMESTAMP"
+	AKSKubernetesVersion                  = "AKS_KUBERNETES_VERSION"
+	SecurityScanFailThreshold             = "SECURITY_SCAN_FAIL_THRESHOLD"
+	SecurityScanContainer                 = "SECURITY_SCAN_CONTAINER"
+	ManagedClustersResourceType           = "managedClusters"
+	capiImagePublisher                    = "cncf-upstream"
+	capiOfferName                         = "capi"
+	capiWindowsOfferName                  = "capi-windows"
+	aksClusterNameSuffix           string = "aks"
```

Comment:
why do we need to string type here but not for others?

Reply:
The best answer I can come up with is some kind of OCD preventing me from not strongly typing the const. But yeah, better to be consistent.